src/info/Info.js

```diff
@@ -1,9 +1,22 @@
-import self from "../img/self.png"
-import mock1 from "../img/mock1.png"
-import mock2 from "../img/mock2.png"
-import mock3 from "../img/mock3.png"
-import mock4 from "../img/mock4.png"
-import mock5 from "../img/mock5.png"
+import self from "../img/self.png";
+import mock1 from "../img/mock1.png";
+import mock2 from "../img/mock2.png";
+import mock3 from "../img/mock3.png";
+import mock4 from "../img/mock4.png";
+import mock5 from "../img/mock5.png";
+import mock_dashboard_ui from "../img/mock_dashboard_ui.png";
+import result_summary_ui from "../img/result_summary_ui.png";
+import ecommerce from "../img/ecommerce.png";
+
+import a from "../img/articles/a.png";
+import b from "../img/articles/b.png";
+import c from "../img/articles/c.png";
+import d from "../img/articles/d.png";
+import e from "../img/articles/e.png";
+import f from "../img/articles/f.png";
+import g from "../img/articles/g.png";
+import h from "../img/articles/h.png";
+import i from "../img/articles/i.png";
```

Comment:
**Suggestion:** Consider using a more descriptive variable name for the imported images. This will make your code more readable and maintainable.
```suggestion
import articleImageA from "../img/articles/a.png";
import articleImageB from "../img/articles/b.png";
import articleImageC from "../img/articles/c.png";
import articleImageD from "../img/articles/d.png";
import articleImageE from "../img/articles/e.png";
import articleImageF from "../img/articles/f.png";
import articleImageG from "../img/articles/g.png";
import articleImageH from "../img/articles/h.png";
import articleImageI from "../img/articles/i.png";
```